addons/vrm/vrm_spring_bone.gd

```diff
@@ -0,0 +1,96 @@
+@tool
+class_name VRMSpringBone
+extends Resource
+
+const VRMSpringBoneLogic = preload("./vrm_spring_bone_logic.gd")
+const vrm_collider_group = preload("./vrm_collider_group.gd")
+const vrm_collider = preload("./vrm_collider.gd")
+
+# Annotation comment
+@export var comment: String
+
+# bone name of the root bone of the swaying object, within skeleton.
+@export var joint_nodes: PackedStringArray
+
+# The resilience of the swaying object (the power of returning to the initial pose).
+@export var stiffness_force: PackedFloat64Array
+# The strength of gravity.
+@export var gravity_power: PackedFloat64Array
+# The direction of gravity. Set (0, -1, 0) for simulating the gravity.
```

Comment:
Double space `##` places documentation in to the help browser. See godot engine docs.